opm/autodiff/WellInterface.hpp

```diff
@@ -64,6 +63,11 @@ namespace Opm
         using WellState = WellStateFullyImplicitBlackoil;
 
         typedef BlackoilModelParameters ModelParameters;
+
+        static const int Water = BlackoilPhases::Aqua;
```

Comment:
Can we use `Water`, `Oil` and `Gas` also for other places instead of `BlackoilPhases::`?

Reply:
maybe it is better to go the whole way and switch this to `FluidSystem::{water,gas,oil}PhaseIdx`...

Reply:
All these indices causing confusions from time to time. It is good to stick to only one when possible. I do not have a preference on which one is better than others.

Reply:
in all the eWoms parts, only the of the names from the `FluidSystem` are available, so we should probably also use those here. (As I see it, the other phase names are only used for historical reasons.)

Reply:
I think it is best to stick to BlackoilPhases:: for the moment since that's what is used in the rest of the well model. I couldn't find any other place where BlackoilPhases is used in the code? Could you point me to where it is?

Reply:
Thanks for the response. Now I can see you moved same definition of `Water`, `Oil` and `Gas` out of the `BlackoilEnums`.  I always get confused by the `Water`, `Oil` and `Gas` and their `BlackoilPhases:` ones.  I can see now the efforts of removing `BlackoilEnums` from Ebos in this PR. It is Okay. Sorry for the noise.